bin/vue-init

```diff
@@ -100,11 +100,15 @@ function run () {
     checkVersion(function () {
       if (!hasSlash) {
         // use official templates
-        template = 'vuejs-templates/' + template
+        var officialTemplate = 'vuejs-templates/' + template
```

Comment:
I like the addition of `officialTemplate`! Makes the code a bit clearer and keeps the value of `template` more predictable as exactly what the user entered. 👍